db/pre-re/item_db_usable.yml

```diff
@@ -1061,7 +1061,7 @@ Body:
       BuyingStore: true
       Container: true
     Script: |
-      getrandgroupitem(IG_BlueBox,1);
+      getgroupitem(IG_BLUEBOX);
```

Comment:
Doesnt that change the "identify" behavior?

getrandgroupitem:
```
item_tmp.identify = identify ? 1 : itemdb_isidentified(entry->nameid);
```

getgroupitem:
```
	bool identify = false;

	if( script_hasdata( st, 3 ) ){
		identify = script_getnum( st, 3 );
	}
```

Reply:
For getrandgroupitem and getgroupitem, identify is defined to false by default. In case of getgroupitem, itemdb_isidentified is handled by `ItemGroupDatabase::pc_get_itemgroup_sub` https://github.com/rathena/rathena/blob/master/src/map/itemdb.cpp#L2964
To resume the "identify" behavior is the same in both commands